lib/HTML/Renderers/SpanNodeRenderer.php

```diff
@@ -69,7 +71,7 @@ public function link(?string $url, string $title, array $attributes = []): strin
 
     public function escape(string $span): string
     {
-        return htmlspecialchars($span);
+        return htmlspecialchars($span, ENT_COMPAT);
```

Comment:
The alternative solution would be to use the new values and adapt the test, but it's kind of a BC-break, so maybe todo on 0.6.x? Let me know what you think.